libdevcore/CommonData.h

```diff
@@ -43,7 +43,7 @@ enum class WhenError
 };
 
 template <class Iterator>
-std::string toHex(Iterator _it, Iterator _end, std::string _prefix)
+std::string toHex(Iterator _it, Iterator _end, const std::string& _prefix)
```

Comment:
We write `std::string const&` not `const std::string&`. Can you change it this way?